components/prism-elixir.js

```diff
@@ -1,5 +1,15 @@
 Prism.languages.elixir = {
-	'comment': /#.*/m,
+	'doc': {
+		pattern: /@(?:doc|moduledoc)\s+("|'|"""|''')(?:\\[\s\S]|(?!\1)[^\\])+\1/,
```

Comment:
Alright, that one should work!

```suggestion
		pattern: /@(?:doc|moduledoc)\s+(?:("""|''')[\s\S]*?\1|("|')(?:\\(?:\r\n|[\s\S])|(?!\2)[^\\\r\n])*\2)/,
```